### PR TITLE
Add PyCharm, notebooks, egg-info to gitignore.

### PR DESCRIPTION
**Pull request by petterbejo**
Thought it might be a good idea to update the gitignore to avoid things getting messy.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
\ No newline at end of file
+*~
+
+# pycharm
+.idea/
+
+.ipynb_checkpoints/
+limnos.egg-info/
+limnos/.ipynb_checkpoints/
+limnos/Untitled.ipynb
```
Comment (WilliamHPNielsen) [inline .gitignore:10]: Why is `Untitled.ipynb` explicitly ignored? Nothing in limnos generates such a file (right?)

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+limnos/__pycache__/
+limnos/tests/__pycache__/
```
Comment (WilliamHPNielsen) [inline .gitignore:12]: It is sufficient to have a single `__pycache__` line; we don't need one for each subfolder.

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,13 @@
...
-*~
\ No newline at end of file
+*~
+
+# pycharm
+.idea/
+
+.ipynb_checkpoints/
+limnos.egg-info/
+limnos/.ipynb_checkpoints/
```
Comment (WilliamHPNielsen) [inline .gitignore:9]: This line is redundant as line 7 already excludes checkpoints.

**Review by WilliamHPNielsen — CHANGES_REQUESTED** [review]
I left some nitpicky inline comments that I think we should address before merging.

**Comment by petterbejo** [comment]
I've tidied the gitignore by removing duplicate pycache and checkpoints folders. I've also removed the Untitled.ipynb file from the gitignore and renamed it Limnos-UI since I consider this being more user friendly (I could have made a separate branch for this, but since it's such a small change and it is somehow related to this one, I've included it here).

**Review by WilliamHPNielsen — APPROVED** [review]
Sweet!

**Pull request merged**